packages/remix-dev/cli/migrate/migrations/replace-remix-imports/transform/options.ts

```diff
@@ -7,7 +7,7 @@
 import type { Adapter } from "./adapter";
 import type { Runtime } from "./runtime";
 
-export interface Options {
-  runtime: Runtime;
+export type Options = {
   adapter?: Adapter;
-}
+  runtime: Runtime;
+};
```

Comment:
I looks like this change doesn't semantically change anything. Can you keep this code as-is?

I personally like having `runtime` and then `adapter` since the `runtime` is required.

Reply:
I needed to make it a `type` instead of `interface`, as `Option` should now extend `Record<string, unknown>`.

If I'm not making it a `type`, I get `TypeError`s